apps/frontend/src/components/layout/Header.tsx

```
@@ -1,77 +1,80 @@
+		<header className="bg-white border-b border-gray-200 py-3 px-4 flex items-center justify-between">
+			<div className="flex items-center space-x-4">
+				<div className="flex items-center">
+					<span className="text-blue-700 font-bold text-2xl">MedLab Chat</span>
+					<span className="text-gray-500 text-xs ml-2">by Angstrom AI</span>
+				</div>
+
+				<div className="relative">
+					<button
+						className="flex items-center space-x-1 bg-gray-100 hover:bg-gray-200 transition-colors px-3 py-1.5 rounded-md text-gray-700"
+						onClick={() => setIsWorkspaceDropdownOpen(!isWorkspaceDropdownOpen)}
+					>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)